samples/GalleryApp/MainPage.xaml

```diff
@@ -9,57 +9,67 @@
+            <RowDefinition Height="10*"/>
         </Grid.RowDefinitions>
 
+        <!--Thumbnails-->
         <Grid Grid.Row="1">
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="40"/>
                 <ColumnDefinition />
-                <ColumnDefinition />
+                <ColumnDefinition Width="40"/>
             </Grid.ColumnDefinitions>
 
-            <!--Skeletal Skill Thumbnail-->
-            <Grid Grid.Column="1" Background="LightGray" Margin="25" Height="400" 
-                  Width="400" HorizontalAlignment="Left" VerticalAlignment="Top">
-                <Grid Height="150" Width="400" VerticalAlignment="Bottom">
-                    <Rectangle Height="200" Width="400" VerticalAlignment="Bottom"
-                                                       Fill="Gray" Opacity="0.75"/>
-                    <TextBlock Text="Skeletal Detector" 
-                               Style="{StaticResource SubheaderTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 0, 0, 0" />
-                    <TextBlock Text="A set of sample apps that can detect and identify the poses of the bodies of individuals in a video or image." 
-                               Style="{StaticResource BodyTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 60, 0, 0"
-                               TextWrapping="Wrap"/>
-                </Grid>
-                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToSkeletalDetectorPage" Style="{StaticResource ButtonRevealStyle}"/>
-            </Grid>
-
-            <Grid Grid.Column="2" Background="LightGray" Margin="25" Height="400" 
-                  Width="400" HorizontalAlignment="Left" VerticalAlignment="Top">
-                <Grid Height="150" Width="400" VerticalAlignment="Bottom">
-                    <Rectangle Height="200" Width="400" VerticalAlignment="Bottom"
-                                                       Fill="Gray" Opacity="0.75"/>
-                    <TextBlock Text="Object Detector" 
-                               Style="{StaticResource SubheaderTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 0, 0, 0" />
-                    <TextBlock Text="A set of sample apps that detect and classify 80 objects using the Object Detector NuGet package." 
-                               Style="{StaticResource BodyTextBlockStyle}"
-                               Foreground="White"
-                               Padding="10, 60, 0, 0"
-                               TextWrapping="Wrap"/>
-                </Grid>
-                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToObjectDetectorPage" Style="{StaticResource ButtonRevealStyle}"/>
-            </Grid>
+            <!--Category bar-->
+            <controls:TabView x:Name="Tabs"
```

Comment:
>TabView [](start = 22, length = 7)

for future you may want to bind SkillCategories directly to this control, right now only SkillViews from Vision will be picked up

Reply:
Task 22514227: Load category tabs in MainPage using template & data binding

---
In reply to: [302149482](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/45#discussion_r302149482) [](ancestors = 302149482)